src/script_chainer/config/script_config.py

```
@@ -36,32 +37,46 @@ class GameProcessName(Enum):
+    def from_dict(cls, data: dict) -> 'ScriptConfig':
+        """从字典反序列化。"""
+        valid = {f.name for f in fields(cls)} - {'idx'}
+        return cls(**{k: v for k, v in data.items() if k in valid})
+
+    @classmethod
+    def create_default(cls) -> 'ScriptConfig':
+        """创建默认配置。"""
+        return cls(check_done=CheckDoneMethods.GAME_OR_SCRIPT_CLOSED.value.value)
+
+    def copy(self) -> 'ScriptConfig':
+        """深拷贝（保留 idx）。"""
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)